mindsdb/integrations/handlers/langchain_handler/langchain_handler.py

```diff
@@ -30,12 +30,13 @@
 # 2 minutes should be more than enough time to complete chains.
 _DEFAULT_AGENT_TIMEOUT_SECONDS = 120
 _DEFAULT_AGENT_MODEL = 'zero-shot-react-description'
-_DEFAULT_AGENT_TOOLS = ['python_repl', 'wikipedia']  # these require no additional arguments
+_DEFAULT_AGENT_TOOLS = ['wikipedia']  # these require no additional arguments
```

Comment:
Curious as to why the Python REPL was removed? Too unstable?

Reply:
It raises error on version =0.0.347: Got unknown tool python_repl
Looks like it[ was removed ](https://github.com/langchain-ai/langchain/blob/v0.0.347/libs/langchain/langchain/agents/load_tools.py#L99)due security issues